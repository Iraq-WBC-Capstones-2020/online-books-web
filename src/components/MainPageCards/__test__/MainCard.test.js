import React from 'react';
import renderer from 'react-test-renderer';
import Cards from './../MainCard';

test('MainCard snapshot', () => {
  const tree = renderer.create(<Cards />).toJSON();
  expect(tree).toMatchSnapshot();
});
