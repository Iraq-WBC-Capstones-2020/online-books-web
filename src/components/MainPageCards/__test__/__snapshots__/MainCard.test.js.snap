// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainCard snapshot 1`] = `
     <div  key={id} className="col-12 col-md-3 col-lg-3 h-90">
      <div
        tag="div"
        data-test="card"
        className="relative h-100 Card shadow-xl animate transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-110"
      >
        <img src={cover} className="imgCardBook rounded-t-md" />

        <div className=" card-body absolute  bottom-0 w-full px-0 py-0"
        data-test="card"
        >
          <div className="">
            <div className="blackOpacity text-center py-1">
              <h5 className="HeaderCardBook mb-0 text-xl text-white">
                {authorName}
              </h5>
              <div className="sizeStarsContainer">
                <CreateStars rate={rate} starSize="small" />
              </div>
              <p className="HeaderCardBook grayColor text-sm font-bold">
                {bookName}
              </p>
            </div>
          
            <div className="bg-black d-flex flex-row justify-content-around py-2 px-2">
              <p className="lightBlue font-semibold font-sans text-lg p-1">
                {newPrice}
              </p>
              <p className="darkGray font-normal line-through p-1">
                {oldPrice}
              </p>

              <div className="rounded-full w-8 h-8 border-2 border-orange-200 text-center">
                <CartIcon className="p-1" />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
`;
